install/data/install_config.txt
a:89:{i:0;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"theme";s:5:"title";s:12:"支持语言";s:7:"varname";s:14:"setup_language";s:6:"varval";s:9:"cn|中文";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:27:"当前主题支持的语言";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:1;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"theme";s:5:"title";s:12:"默认语言";s:7:"varname";s:18:"setup_language_def";s:6:"varval";s:2:"cn";s:7:"vartype";s:1:"1";s:8:"infotype";s:6:"select";s:7:"infosel";s:0:"";s:7:"content";s:45:"首次打开网站显示的默认语言版本";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"2";}i:2;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"theme";s:5:"title";s:12:"伪静态URL";s:7:"varname";s:12:"setup_static";s:6:"varval";s:5:"false";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:66:"true|开启（服务器需支持伪静态URL规则）,false|关闭";s:7:"content";s:93:"以静态方式显示URL，需服务器支持伪静态规则，设计的主题需符合要求";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"5";}i:3;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"setup";s:5:"title";s:18:"留言接收邮箱";s:7:"varname";s:16:"setup_smtp_email";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:145:"当网站中收到留言信息时会自动发送到此邮箱，留空表示不需要发送邮件，此功能需要设置好SMTP服务器才有效";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:4;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"setup";s:5:"title";s:18:"临时文件提醒";s:7:"varname";s:18:"setup_tip_tempfile";s:6:"varval";s:1:"5";s:7:"vartype";s:1:"2";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:41:"临时文件达到多少MB时提醒清理";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"3";}i:5;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"smtp";s:5:"title";s:13:"smtp服务器";s:7:"varname";s:17:"setup_smtp_server";s:6:"varval";s:11:"smtp.qq.com";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:14:"如smtp.qq.com";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:6;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"smtp";s:5:"title";s:10:"smtp端口";s:7:"varname";s:15:"setup_smtp_port";s:6:"varval";s:2:"25";s:7:"vartype";s:1:"2";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:11:"通常为25";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:7;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"smtp";s:5:"title";s:12:"登录账号";s:7:"varname";s:15:"setup_smtp_user";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:8;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"smtp";s:5:"title";s:12:"登录密码";s:7:"varname";s:15:"setup_smtp_pass";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:4:"pass";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:9;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"smtp";s:5:"title";s:12:"发送邮箱";s:7:"varname";s:15:"setup_smtp_from";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:49:"必须是前面设置的smtp服务器上的邮箱";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:10;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"smtp";s:5:"title";s:18:"邮件附加提示";s:7:"varname";s:16:"setup_smtp_fooer";s:6:"varval";s:93:"本邮件由系统自动发送，您无需回复本邮件，如有疑问请咨询工作人员";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:49:"发送的邮件底部附加内容，不支持HTML";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:11;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:15:"编缉器水印";s:7:"varname";s:14:"setup_marktype";s:6:"varval";s:5:"false";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:60:"是否对文章编缉器中上传的图片开启水印功能";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:12;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:12:"水印图片";s:7:"varname";s:13:"setup_markimg";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:6:"upload";s:7:"infosel";s:0:"";s:7:"content";s:27:"建议为png-24格式图片";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"3";}i:13;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:15:"水印透明度";s:7:"varname";s:15:"setup_markalpha";s:6:"varval";s:2:"40";s:7:"vartype";s:1:"2";s:8:"infotype";s:6:"select";s:7:"infosel";s:97:"10|10%（最透明）,20|20%,30|30%,40|40%,50|50%,60|60%,70|70%,80|80%,90|90%,100|完全不透明";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"4";}i:14;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:12:"水印位置";s:7:"varname";s:12:"setup_markxy";s:6:"varval";s:1:"3";s:7:"vartype";s:1:"2";s:8:"infotype";s:6:"select";s:7:"infosel";s:122:"1|左上角,2|上居中,3|右上角,4|左垂直居中,5|完全居中,6|右垂直居中,7|左下角,8|下居中,9|右下角";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"5";}i:15;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:18:"水印距离边距";s:7:"varname";s:14:"setup_markside";s:6:"varval";s:2:"20";s:7:"vartype";s:1:"2";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:65:"单位：像素，建议0~100，水印与原图片的边距大小";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"6";}i:16;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:18:"不加水印尺寸";s:7:"varname";s:13:"setup_markmin";s:6:"varval";s:3:"200";s:7:"vartype";s:1:"2";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:63:"单位：象素，图片高或宽低于此尺寸时不打水印";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"7";}i:17;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:21:"单文件大小上限";s:7:"varname";s:12:"setup_upsize";s:6:"varval";s:5:"10240";s:7:"vartype";s:1:"2";s:8:"infotype";s:6:"select";s:7:"infosel";s:109:"200|200 kb,500|500 kb,1024|1 mb,2048|2 mb,5120|5 mb（推荐）,8192|8 mb,10240|10 mb,15360|15 mb,20480|20 mb";s:7:"content";s:56:"允许上传文件的最大值，建议在2～10mb之间";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"8";}i:18;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:6:"upload";s:5:"title";s:15:"gif原图上传";s:7:"varname";s:11:"setup_upgif";s:6:"varval";s:4:"true";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:120:"如果关闭，则gif将以jpg静态图片文式保存，如果开启，建议服务器禁止上传主目录运行脚本";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:2:"13";}i:19;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:18:"侧栏工具开关";s:7:"varname";s:10:"setup_tool";s:6:"varval";s:5:"false";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:60:"true|开启（网站侧边工具栏总开关）,false|关闭";s:7:"content";s:33:"网站右侧漂浮工具总开关";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:20;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:14:"QQ在线客服";s:7:"varname";s:19:"setup_tool_customer";s:6:"varval";s:4:"true";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"2";}i:21;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:12:"在线留言";s:7:"varname";s:19:"setup_tool_feedback";s:6:"varval";s:4:"true";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"4";}i:22;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:12:"联系电话";s:7:"varname";s:14:"setup_tool_tel";s:6:"varval";s:4:"true";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"5";}i:23;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:12:"电话号码";s:7:"varname";s:18:"setup_tool_tel_num";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:72:"设置在工具栏中显示的电话号码，多个用英文逗号分隔";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"6";}i:24;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:15:"微信二维码";s:7:"varname";s:17:"setup_tool_weixin";s:6:"varval";s:4:"true";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"7";}i:25;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:15:"微信公众号";s:7:"varname";s:21:"setup_tool_weixin_num";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:48:"设置在工具栏显示的微信公众号账号";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"8";}i:26;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:4:"tool";s:5:"title";s:18:"公众号二维码";s:7:"varname";s:21:"setup_tool_weixin_img";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:6:"upload";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"9";}i:27;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"theme";s:5:"title";s:21:"智能切换移动端";s:7:"varname";s:12:"setup_mobile";s:6:"varval";s:5:"false";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:352:"1、开启时，如果是移动端访问网站，将会自动读取手机模板，假设当前主题为中文版（cn目录），如果是pc端访问，会加载cn目录下的模板，如果是移动端，则加载cn_mobie目录下的模板&lt;br&gt;2、如果主题模板为响应式设计，支持自适应PC及移动端的分辨率，这里请关闭";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"3";}i:28;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:18:"网站首页名称";s:7:"varname";s:13:"setup_webname";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:50:"完整网站名称，用于网站首页title标签";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:29;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:18:"网站内页名称";s:7:"varname";s:18:"setup_webname_page";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:72:"网站除首页外的标题，前面将会自动加上当前页面标题";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:30;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"网站简称";s:7:"varname";s:15:"setup_shortname";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:51:"简短的网站名称，用于特定的地方显示";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:31;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"SEO关键词";s:7:"varname";s:13:"setup_keyword";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:4:"text";s:7:"infosel";s:0:"";s:7:"content";s:64:"用于meta keywords标签，多个关键词用英文逗号分隔";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:32;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"网站描述";s:7:"varname";s:17:"setup_description";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:4:"text";s:7:"infosel";s:0:"";s:7:"content";s:85:"用于meta description标签，通常在200个字以内，完整的网站描述语句";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:33;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"公司名称";s:7:"varname";s:13:"setup_company";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:34;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"公司邮箱";s:7:"varname";s:19:"setup_company_email";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:35;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"公司电话";s:7:"varname";s:17:"setup_company_tel";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:36;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"公司传真";s:7:"varname";s:17:"setup_company_fax";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:37;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:3:"web";s:5:"title";s:12:"公司地址";s:7:"varname";s:21:"setup_company_address";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:38;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"客服信息分组";s:7:"varname";s:19:"setup_customer_bcat";s:6:"varval";s:43:"未归类,QQ客服,Skype客服,其它客服";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:27:"多个用英文逗号分隔";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:39;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"错误标签提示";s:7:"varname";s:14:"setup_errortag";s:6:"varval";s:19:"[!--标签错误--]";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:48:"当模板中插入错误标签时返回的信息";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:40;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:12:"分页标签";s:7:"varname";s:13:"setup_pagetag";s:6:"varval";s:18:"{article_page_tag}";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:78:"文章编缉器中插入的分页符，不同与主题模板中的分页标签";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:41;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"主题安装文件";s:7:"varname";s:15:"setup_themefile";s:6:"varval";s:14:"theme_data.txt";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:42;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"setup";s:5:"title";s:18:"模板文件缓存";s:7:"varname";s:15:"setup_tempcache";s:6:"varval";s:5:"false";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:126:"模板文件将会缓存，加快系统运行速度，但开启后每次修改了主题，都要清除网站缓存才会生效";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"5";}i:43;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"缓存文件后缀";s:7:"varname";s:15:"setup_cachetype";s:6:"varval";s:4:"html";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:44;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"setup";s:5:"title";s:18:"静态页面缓存";s:7:"varname";s:15:"setup_datacache";s:6:"varval";s:1:"0";s:7:"vartype";s:1:"2";s:8:"infotype";s:6:"select";s:7:"infosel";s:130:"0|不启用,10|10分钟,30|30分钟,60|1小时,180|3小时,480|8小时,720|12小时,1440|1天,10080|1周,43200|1个月,525600|1年";s:7:"content";s:123:"开启此项相当于生成静态页面，可以提升页面载入速度，如启用则表示多长时间自动更新缓存";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"7";}i:45;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"模板文件后缀";s:7:"varname";s:14:"setup_temptype";s:6:"varval";s:4:"html";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:46;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"模板标签前缀";s:7:"varname";s:12:"setup_prefix";s:6:"varval";s:2:"{:";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:47;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"模板标签后缀";s:7:"varname";s:12:"setup_suffix";s:6:"varval";s:1:"}";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:48;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:24:"防止调用文件缓存";s:7:"varname";s:14:"setup_filerand";s:6:"varval";s:6:"time()";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:64:"0|正常缓存,time()|禁止文件缓存,date()|缓存期限1天";s:7:"content";s:80:"在网站页面中调用的css、js等文件后面加上随机数，防止缓存";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:49;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:21:"后台每页记录数";s:7:"varname";s:13:"setup_am_page";s:6:"varval";s:2:"20";s:7:"vartype";s:1:"2";s:8:"infotype";s:6:"select";s:7:"infosel";s:30:"10,20|20（推荐）,30,50,100";s:7:"content";s:42:"后台分页列表每页显示的记录数";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:50;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"后台高级功能";s:7:"varname";s:13:"setup_am_more";s:6:"varval";s:4:"true";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:24:"true|开启,false|关闭";s:7:"content";s:36:"后台是否开启高级管理功能";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:51;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:24:"后台维护密码开关";s:7:"varname";s:14:"setup_am_super";s:6:"varval";s:5:"false";s:7:"vartype";s:1:"3";s:8:"infotype";s:6:"select";s:7:"infosel";s:27:"false|不启用,true|启用";s:7:"content";s:93:"是否启用后台维护密码功能，通过后台维护密码可登录任意管理员账号";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:52;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"设置维护密码";s:7:"varname";s:19:"setup_am_super_pass";s:6:"varval";s:0:"";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:47:"设置一个维护密码，请输入32位的md5";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:53;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"系统设置分类";s:7:"varname";s:19:"setup_am_setup_cata";s:6:"varval";s:93:"setup|系统综合,tool|侧栏工具,upload|上传设置,smtp|邮件设置,theme|主题设置";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:0:"";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:54;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:15:"模板主目录";s:7:"varname";s:15:"setup_webfolder";s:6:"varval";s:9:"template/";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:30:"站点模板存放的主目录";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"1";s:6:"isview";s:1:"1";s:4:"sort";s:1:"1";}i:55;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:15:"上传主目录";s:7:"varname";s:14:"setup_upfolder";s:6:"varval";s:7:"upload/";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:101:"文件上传主目录，建议设置此目录为不可运行脚本，以 / 结尾，开头不用加 /";s:7:"contype";s:1:"0";s:6:"isedit";s:1:"0";s:6:"isview";s:1:"0";s:4:"sort";s:1:"1";}i:56;a:13:{s:5:"webid";s:1:"1";s:4:"cata";s:5:"basic";s:5:"title";s:18:"临时文件目录";s:7:"varname";s:12:"setup_uptemp";s:6:"varval";s:5:"temp/";s:7:"vartype";s:1:"1";s:8:"infotype";s:3:"inp";s:7:"infosel";s:0:"";s:7:"content";s:54:"此目录位于上传主目录内，存放临时文件";s:7:"contype";

include/config.php
<?php
include('config_db.php');
include('config_qyk.php');
/*----基础配置更新于2015-11-24 23:52:00----*/
define('setup_customer_bcat','未归类,QQ客服,Skype客服,其它客服');
define('setup_errortag','[!--标签错误--]');
define('setup_pagetag','{article_page_tag}');
define('setup_themefile','theme_data.txt');
define('setup_cachetype','html');
define('setup_temptype','html');
define('setup_prefix','{:');
define('setup_suffix','}');
define('setup_filerand',time());
define('setup_am_page',20);
define('setup_am_more',true);
define('setup_am_super',false);
define('setup_am_super_pass','');
define('setup_am_setup_cata','setup|系统综合,tool|侧栏工具,upload|上传设置,smtp|邮件设置,theme|主题设置');
define('setup_webfolder','template/');
define('setup_upfolder','upload/');
define('setup_uptemp','temp/');
define('setup_uploadtag','{upload_path_tag}');
define('setup_editfile_bak',false);
define('setup_static_file','httpd.ini');
define('setup_theme_artnum',1000);
define('setup_login_error_num',5);
define('setup_login_error_time',600);
define('setup_tempcache_md5',false);
define('setup_websetup_del',true);
?>